src/_guides/language/language-tour.md

```diff
@@ -1697,36 +1709,33 @@ When you need to assign a value
 based on a boolean expression,
 consider using `?:`.
 
-<!-- library-tour/mirrors/bin/main.dart -->
+<?code-excerpt "lib/language_tour/operators.dart (if-then-else-operator)"?>
 {% prettify dart %}
-var finalStatus = m.isFinal ? 'final' : 'not final';
+var visibility = isPublic ? 'public' : 'private';
 {% endprettify %}
 
 If the boolean expression tests for null,
 consider using `??`.
 
+<?code-excerpt "test/language_tour/operators_test.dart (if-null)" replace="/\d//g"?>
 {% prettify dart %}
-String toString() => msg ?? super.toString();
+String playerName(String name) => name ?? 'Guest';
 {% endprettify %}
 
 The previous example could have been written at least two other ways,
 but not as succinctly:
 
-{% comment %}
-https://dartpad.dartlang.org/f2c8d82ce5d0dd533fe2
-https://gist.github.com/f2c8d82ce5d0dd533fe2
-{% endcomment %}
-
+<?code-excerpt "test/language_tour/operators_test.dart (if-null-alt)" replace="/\d//g"?>
 {% prettify dart %}
 // Slightly longer version uses ?: operator.
-String toString() => msg == null ? super.toString() : msg;
+String playerName(String name) => name != null ? name : 'Guest';
```

Comment:
Just a reminder that offline, we had agreed to replace the old example using `super.toString()` by a simpler one that doesn't make use of `super`.